WalletWasabi.Fluent.Desktop/WalletWasabi.Fluent.Desktop.csproj

```diff
@@ -57,4 +61,8 @@
 		<ProjectReference Include="..\WalletWasabi.Gui\WalletWasabi.Gui.csproj" />
 		<ProjectReference Include="..\WalletWasabi\WalletWasabi.csproj" />
 	</ItemGroup>
+
+	<ItemGroup>
+		<RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
```

Comment:
What is this for?

Reply:
Without this exception, messages would be localized to the system's language.